src/constants.ts

```diff
@@ -12,6 +12,7 @@ export const EXCLUDED_NAMES = [GIT_FILENAME, GITIGNORE_FILENAME, DCIGNORE_FILENA
 export const CACHE_KEY = '.dccache';
 export const MAX_UPLOAD_ATTEMPTS = 5;
 export const UPLOAD_CONCURRENCY = 5;
+export const POLLING_INTERVAL = 1000;
```

Comment:
question: looks like polling interval was changed from 500 to 1000. Any reason for that?

Reply:
ah, probably i did it by mistake, I thought it was 1 sec.
There was not much thinking on choosing it in the first place too.
I guess i can revert it back to 500ms

Reply:
done